packages/client/src/components/pages/muster-page/table-unit-muster-compliance.tsx

```diff
@@ -155,6 +119,57 @@ export const UnitMusterComplianceTable = (props: any) => {
   // Functions
   //
 
+  const getUnitComplianceData = async (organizationId: number, unitName: string, isoStartDate: string, isoEndDate: string): Promise<ComplianceData> => {
+    const xRange: string[] = [];
+    const yRange: number[] = [];
+    const colors: string[] = [];
+    let dayCount: number = 0;
+    let totalCompliance: number = 0;
```

Comment:
I think it's fine in cases like these to just let TS infer the type. We've generally been using type inference where possible up to this point.

```suggestion
    let dayCount = 0;
    let totalCompliance = 0;
```